src/clj/jsonista/core.clj

```diff
@@ -259,3 +335,53 @@
    (-write-value to object default-object-mapper))
   ([to object ^ObjectMapper mapper]
    (-write-value to object mapper)))
+
+(defn- wrap-values
+  [^Iterator iterator]
+  (when iterator
+    (reify
+      Iterable
```

Comment:
One thing I'm unsure of is implementing Iterable here. Is it a good idea to return an object which is both an `Iterable` and `Iterator`?